deploy/manifests/kip/overlays/mac-tests/provider.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+cloud:
+  aws:
+    vpcID: <FILL-IN>
+    region: <FILL-IN>
+    subnetID: <FILL-IN>
+    accessKeyID: <FILL-IN>
+    secretAccessKey: <FILL-IN>
```

Comment:
Shouldn't we remove these and let kip use defaults and its VPC/subnet? Were you testing it via minikube?

Reply:
Good point, I tested on the cloud and had those set in overlay. Gonna remove those from overlay.